### PR TITLE
Update office-365-encryption-risks-and-protections.md

### PR DESCRIPTION
**Pull request by MeszarosEtele**
Updated: Exchange Online TLS certificate information

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/compliance/office-365-encryption-risks-and-protections.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ Some risk scenarios and the currently available encryption technologies that mit
...
-| SMTP TLS with partner organization | Exchange Online | TLS 1.2 with AES 256 | The TLS certificate for Exchange Online (outlook.office.com) is a 2048-bit SHA256RSA certificate issued by Baltimore CyberTrust Root. <br> <br> The TLS root certificate for Exchange Online is a 2048-bit SHA1RSA certificate issued by Baltimore CyberTrust Root. | Yes, when TLS 1.2 with 256-bit cipher strength is used |
+| SMTP TLS with partner organization | Exchange Online | TLS 1.2 with AES 256 | The TLS certificate for Exchange Online (outlook.office.com) is a 2048-bit SHA-256 with RSA Encryption certificate issued by DigiCert Cloud Services CA-1. <br> <br> The TLS root certificate for Exchange Online is a 2048-bit SHA-1 with RSA Encryption certificate issued by [GlobalSign Root CA – R1](https://docs.microsoft.com/en-us/microsoft-365/compliance/exchange-online-uses-tls-to-secure-email-connections?view=o365-worldwide#tls-certificate-information-for-exchange-online). <br> <br> Be aware that, for security reasons, our certificates do change from time to time. | Yes, when TLS 1.2 with 256-bit cipher strength is used |
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/office-365-encryption-risks-and-protections.md]:
```suggestion
| SMTP TLS with partner organization | Exchange Online | TLS 1.2 with AES 256 | The TLS certificate for Exchange Online (outlook.office.com) is a 2048-bit SHA-256 with RSA Encryption certificate issued by DigiCert Cloud Services CA-1. <br> <br> The TLS root certificate for Exchange Online is a 2048-bit SHA-1 with RSA Encryption certificate issued by [GlobalSign Root CA – R1](https://docs.microsoft.com/microsoft-365/compliance/exchange-online-uses-tls-to-secure-email-connections?view=o365-worldwide#tls-certificate-information-for-exchange-online). <br> <br> Be aware that, for security reasons, our certificates do change from time to time. | Yes, when TLS 1.2 with 256-bit cipher strength is used |
```

File: microsoft-365/compliance/office-365-encryption-risks-and-protections.md [hunk 2]
```diff
@@ -83,6 +83,6 @@ Some risk scenarios and the currently available encryption technologies that mit
...
-| SMTP TLS with partner organization | Exchange Online | TLS 1.2 with AES 256 | The TLS certificate for Exchange Online (outlook.office.com) is a 2048-bit SHA256RSA certificate issued by Baltimore CyberTrust Root. <br> <br> The TLS root certificate for Exchange Online is a 2048-bit sha1RSA certificate issued by Baltimore CyberTrust Root. <br> <br> Be aware that, for security reasons, our certificates do change from time to time. | Yes |
+| SMTP TLS with partner organization | Exchange Online | TLS 1.2 with AES 256 | The TLS certificate for Exchange Online (outlook.office.com) is a 2048-bit SHA-256 with RSA Encryption certificate issued by DigiCert Cloud Services CA-1. <br> <br> The TLS root certificate for Exchange Online is a 2048-bit SHA-1 with RSA Encryption certificate issued by [GlobalSign Root CA – R1](https://docs.microsoft.com/en-us/microsoft-365/compliance/exchange-online-uses-tls-to-secure-email-connections?view=o365-worldwide#tls-certificate-information-for-exchange-online). <br> <br> Be aware that, for security reasons, our certificates do change from time to time. | Yes, when TLS 1.2 with 256-bit cipher strength is used |
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/office-365-encryption-risks-and-protections.md]:
```suggestion
| SMTP TLS with partner organization | Exchange Online | TLS 1.2 with AES 256 | The TLS certificate for Exchange Online (outlook.office.com) is a 2048-bit SHA-256 with RSA Encryption certificate issued by DigiCert Cloud Services CA-1. <br> <br> The TLS root certificate for Exchange Online is a 2048-bit SHA-1 with RSA Encryption certificate issued by [GlobalSign Root CA – R1](https://docs.microsoft.com/microsoft-365/compliance/exchange-online-uses-tls-to-secure-email-connections?view=o365-worldwide#tls-certificate-information-for-exchange-online). <br> <br> Be aware that, for security reasons, our certificates do change from time to time. | Yes, when TLS 1.2 with 256-bit cipher strength is used |
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - please remove the locale from the links.

File: microsoft-365/compliance/office-365-encryption-risks-and-protections.md (hunk 2 above)
Comment (MeszarosEtele) [reply microsoft-365/compliance/office-365-encryption-risks-and-protections.md]: I removed the locale from the links.

File: microsoft-365/compliance/office-365-encryption-risks-and-protections.md (hunk 1 above)
Comment (MeszarosEtele) [reply microsoft-365/compliance/office-365-encryption-risks-and-protections.md]: I removed the locale from the links.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by MeszarosEtele** [comment]
@JohanFreelancer9 : I removed the locale from the links and commited changes.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kelleyvice-msft Please merge this PR. Thanks.

**Pull request merged**